borg/helpers.py

```diff
@@ -348,6 +348,33 @@ def __str__(self):
         return self.pattern_orig
 
 
+class ExcludeRegex(IncludePattern):
+    """Regular expression to exclude.
+    """
+    def __init__(self, pattern):
+        self.pattern_orig = pattern
+        self.match_count = 0
+
+        if sys.platform in ('darwin',):
+            pattern = unicodedata.normalize("NFD", pattern)
+
+        self.pattern = pattern
+        self.regex = re.compile(pattern)
+
+    @normalized
+    def match(self, path):
+        matches = self.regex.search(path) is not None
```

Comment:
hmm, ok, it is more flexible with search. just the inconsistency might confuse people.